lisa/operating_system.py

```diff
@@ -1034,6 +1034,24 @@ def _package_exists(self, package: str) -> bool:
 
         return False
 
+    def install_epel(self) -> None:
```

Comment:
What's the epel for? Why we need a special logic to install it?

Reply:
it's the 'extra packages for enterprise linux' that fedora provides by default but other distros need to add explicitly. It contains a bunch of useful packages not specifically supported by RedHat but that are very commonly used. It's a requirement for many other programs so adding an add function for it makes sense.

Reply:
ok, please add it to the comment of this method.